src/dfilemaker/dfilemaker.c

```diff
@@ -584,6 +629,21 @@ int main(int narg, char **arg)
     nlevels = atoi(arg[2]);
     maxflen = atoi(arg[3]);
 
+    /*------------------------------------------
+    /*  create directory name compare function
+    /*------------------------------------------*/
+    //jll if (DTCMP_Op_create(MPI_DOUBLE,&dnamcomp,&op_dnamcomp) !=DTCMP_SUCCESS)
+    if (DTCMP_Op_create(dirname_type,&dnamcomp,&op_dnamcomp) !=DTCMP_SUCCESS)
```

Comment:
Oh, and this will have to move down as well so that it still comes after committing dirname_type.